nesta/production/luigihacks/elasticsearchplus.py

```diff
@@ -403,7 +538,18 @@ def __init__(self, entity_type,
         if caps_to_camel_case:
             self.transforms.append(_caps_to_camel_case)
 
+        # Translate any text to english
+        if auto_translate:
+            urls = list(f"translate.google.{x}"
+                        for x in ('com', 'co.uk', 'co.kr',
```

Comment:
it would be worth clarifying these urls are just for load balancing

Reply:
Yep!